servers/rendering/renderer_rd/shaders/effects/copy.glsl

```diff
@@ -241,12 +241,18 @@ void main() {
 
 #endif
 
-#ifdef MODE_LINEARIZE_DEPTH_COPY
+#if defined(MODE_LINEARIZE_DEPTH_COPY) || defined(MODE_LINEARIZE_DEPTH_COPY_INVERSED)
 
 	float depth = texelFetch(source_color, pos + params.section.xy, 0).r;
-	depth = depth * 2.0 - 1.0;
-	depth = 2.0 * params.camera_z_near * params.camera_z_far / (params.camera_z_far + params.camera_z_near - depth * (params.camera_z_far - params.camera_z_near));
+
+	depth = params.camera_z_near * params.camera_z_far /
+			(params.camera_z_far + depth * (params.camera_z_near - params.camera_z_far));
```

Comment:
Could you explain in more detail what is wrong with this way of linearizing depth? This is the same method used throughout the engine, so if it changes here it should change there as well

Reply:
I think I was a bit quick in editing that shader, I've restored line 248 in a commit I'll be pushing.
In terms of line 247, if I have understood the maths correctly that sort of conversion should not be needed in Vulkan compared to OpenGL, is it correct that it is no longer needed?

Reply:
No, it is still needed. We aren't dealing with NDC values here, we are dealing with the contents of the depth buffer which are still values in the 0-1 range regardless of using Vulkan or OpenGL. 

If you look at other parts of the codebase, they all use the same function to convert depth buffer depth into linear depth. I would stick with using the same code here as elsewhere so that everything matches.